docs/src/topics/omi.md

```diff
@@ -11,7 +11,10 @@ New OMI will be builded and pushblish every two weeks for each supported Os Dist
 ## Current available image:
 
 Ubuntu:
-- ubuntu-2004-2004-kubernetes-v1.21.14-2022-09-06
+- ubuntu-2004-2004-kubernetes-v1.21.14-2022-10-10
+- ubuntu-2004-2004-kubernetes-v1.22.11-2022-10-10
 - ubuntu-2004-2004-kubernetes-v1.22.11-2022-08-22
-- ubuntu-2004-2004-kubernetes-v1.23.8-2022-08-25
-- ubuntu-2004-2004-kubernetes-v1.24.2-2022-08-22
+- ubuntu-2004-2004-kubernetes-v1.23.8-2022-10-10
+- ubuntu-2004-2004-kubernetes-v1.24.2-2022-10-10
+- ubuntu-2004-2004-kubernetes-v1.25.2-2022-10-10
```

Comment:
It is a good thing to add these OMIs here. 
Could we add a job in the Github Action of `image-builder` to auto update this doc ?